README.md

```diff
@@ -1 +1,3 @@
 # git-course-example
```

Comment:
**_i find this section vague. Please consider adding verbal description of what these commands are actually doing._**